src/features/NewOrder/NewOrder.js

```diff
@@ -0,0 +1,300 @@
+          <div className={styles.error}>
+            You don't have enough cash remaining
+          </div>
+        )}
+        <div className={styles.tally}>
+          {' '}
+          Cash Remaining: ${cashRemaining.toFixed(2)}
+        </div>
+      </div>
+
+      <input type="submit" className={styles.submit} />
+    </form>
+  );
+}
+
+function SellForm() {
+  // variables
+  const { register, handleSubmit, watch, reset, errors } = useForm({
+    mode: 'onBlur',
+    reValidateMode: 'onBlur',
+  });
+
+  const onSubmit = (data) => {
+    console.log(data);
+    reset();
+  };
+
+  const onErrors = (errors) => console.error(errors);
+
+  const stock = useRef({});
+  stock.current = watch('stock');
+
+  const sharePrice = useRef({});
+  sharePrice.current = watch('price', '0.00');
+
+  const quantity = useRef({});
+  quantity.current = watch('quantity', '');
+
+  let totalPrice = sharePrice.current * quantity.current;
+
+  // functions
+  let optionsArray = [];
+  function options() {
+    for (var i = 0; i < ownedStocks.length; i++) {
+      optionsArray[i] = <option>{ownedStocks[i].stockName}</option>;
+    }
+  }
+  options();
+
+  function quantityCheck() {
+    for (var i = 0; i < ownedStocks.length; i++) {
+      if (ownedStocks[i].stockName === stock.current) {
+        return ownedStocks[i].stockShares;
+      } else {
+        return 999999;
+      }
+    }
+  }
+
+  return (
+    <form
+      className={styles.subFormContainer}
+      onSubmit={handleSubmit(onSubmit, onErrors)}
+    >
+      <div>
+        <input name="type" ref={register} defaultValue="sell" hidden />
+        {errors.stock && errors.stock.type === 'validate' && (
+          <div className={styles.error}>Select a stock</div>
+        )}
+        <label htmlFor="stock">Stock To Sell: </label>
+        <select
+          name="stock"
+          className={errors.stock ? styles.selectError : styles.select}
+          ref={register({
+            required: true,
+            validate: (value) => value !== '-select-',
+          })}
+        >
+          <option disabled selected>
+            -select-
+          </option>
+          {optionsArray}
+        </select>
+        <div>
+          {errors.quantity && errors.quantity.type === 'required' && (
+            <div className={styles.error}>Quantity must be selected</div>
+          )}
+          {errors.quantity && errors.quantity.type === 'validate' && (
+            <div className={styles.error}>
+              You have exceeded your owned shares
+            </div>
+          )}
+          <label htmlFor="quantity">Quantity: </label>
+
+          <input
+            className={errors.quantity ? styles.inputError : styles.input}
+            name="quantity"
+            min="1"
+            type="number"
+            step="1"
+            ref={register({
+              required: true,
+              validate: (value) => value <= quantityCheck(),
```

Comment:
I think your quantity check is a great solution. Here's an alternative you may consider for the future:
```
const quantityCheck = (value) =>
                value <=
                  ownedStocks.find((s) => s.stockName === stock.current)
                    ?.stockShares ?? Infinity;

// ... other code here 

validate: quantityCheck
```